metadata-editor/app/store/CollectionsStore.scala

```diff
@@ -0,0 +1,59 @@
+package store
+
+import com.gu.mediaservice.lib.collections.CollectionsManager
+import com.gu.mediaservice.lib.store.JsonStore
+import com.gu.mediaservice.model.Collection
+import play.api.libs.json.{JsValue, Json}
+
+import lib.Config
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+object CollectionsStore {
+  import Config.{awsCredentials, collectionsBucket}
+  val store = new JsonStore(collectionsBucket, awsCredentials, "collections")
+
+  def getAll: Future[List[Collection]] = store.getData flatMap { json =>
+    json.asOpt[List[Collection]].map(Future.successful) getOrElse Future.failed(InvalidCollectionJson(json))
+  }
+
+  def add(collection: Collection): Future[Collection] = {
+    store.getData flatMap { json =>
```

Comment:
:bike::house: until we need to, hopefully the abstraction of the CollectionStore will make it easy to swap out. this will work until we need to be a little more prod ready.